Makefile

```diff
@@ -24,8 +24,6 @@ translations: $(WEBPACK_TEST)
 install: all
 	mkdir -p /usr/share/cockpit/composer
 	cp -r public/* /usr/share/cockpit/composer
-	mkdir -p /usr/share/metainfo/
-	cp io.weldr.cockpit-composer.metainfo.xml /usr/share/metainfo/
```

Comment:
Isn't this required? How is cockpit-composer exposed now?

Reply:
Under `dist` there is a `manifest.json` file which declares cockpit-composers placement on the dashboard. This metainfo file did not seem to be doing anything. It was attached to the applications section of cockpit but, at least locally, did not have any effect.

Reply:
It's used in the applications tool in cockpit. I'd definitely check with them before removing it.

Without it Image Builder won't actually show up under /apps.

Reply:
I will reach out to them. Does installing image-builder from cockpit work for you? It's broken for me. I created this tracker #1578. To reimplement the applicaitons list for cockpit composer.

Reply:
I think you can only ever remove an application from the /apps page.

Reply:
Ok. I readded it in but not in the makefile since the install script has been removed. It is in the specfile.